0115-redefining-the-mobile-maker-approval-process.md

```diff
@@ -84,13 +87,15 @@ In some instances, MOBILE Makers may wish to stop producing MOBILE devices, and
  
   Ultimately, the Helium Foundation will have the final say in determining whether the Maker has satisfied the above. 
 
-The foundation will then require the wallet be replenished so at the time of the audit, the Maker Escrow Wallet holds at least enough tokens so that all remaining devices that have not been onboarded may be onboarded as of that date, plus an extra 1% to cover any swapping fees. For example, if a Maker has a total of 1000 indoor Wi-Fi access points that have yet to be onboarded, they will require to have at least 20,200 USDC within their Maker Escrow Wallet.
+The foundation will then require the onboarding wallet be replenished so at the time of the audit, the Maker onboarding Wallet holds at least enough tokens so that all remaining devices that have not been onboarded may be onboarded as of that date, plus an extra 1% to cover any swapping fees. For example, if a Maker has a total of 1000 indoor Wi-Fi access points that have yet to be onboarded, they will require to have at least 20,200 USDC within their Maker Escrow Wallet.
```

Comment:
```suggestion
The foundation will then require the onboarding wallet be replenished so at the time of the audit, the Maker Onboarding Wallet holds at least enough tokens so that all remaining devices that have not been onboarded may be onboarded as of that date, plus an extra 1% to cover any swapping fees. For example, if a Maker has a total of 1000 indoor Wi-Fi access points that have yet to be onboarded, they will require to have at least 20,200 USDC within their Maker Escrow Wallet.
```